### DISCUSSION TITLE
Treatment Events and Conserv. Descriptions for Preparations

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5809] [Feature Requests]
> **Soraya:**
> Hi. Would it be possible to link the Treatment Event to the Preparation and not only to the Collection Object? If a specimen has 2 preparations, for example a skin and a skeleton, they would need different treatments. it would be possible to create 2 Treatment Events for the Collection Object and for each one indicate the preparation it corresponds to, but it would be clearer to have all the information related to a preparation accessible via the Preparation view.
> **Niels:**
> I second this. Same for Conserv. Description.

**Requested By:** [NielsKlazenga](https://discourse.specifysoftware.org/t/treatment-event-for-preparation/1181/2?u=specify) and [sorovi](https://discourse.specifysoftware.org/t/treatment-event-for-preparation/1181?u=specify) on the [Speciforum](https://discourse.specifysoftware.org/t/treatment-event-for-preparation/1181)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/treatment-event-for-preparation/1181/3

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8218
